src/wasm-binary.h

```diff
@@ -155,7 +155,8 @@ enum Section {
   Functions = 2,
   DataSegments = 4,
   FunctionTable = 5,
-  End = 6
+  End = 6,
+  WLLSegment = 0x11
```

Comment:
I see. I would be ok then to ignore unknown sections, which should give you what you want here, without hardcoding a nonstandard section.

We should print a warning on skipping unknown sections (in `debug` mode).